content/news/2021-XX-XX-fsf-statement.md

```diff
@@ -0,0 +1,25 @@
+title: "A Statement on Richard Stallman and the Free Software Foundation"
+authors: Owen Williams
+tags: community, diversity, politics
+date: 2021-03-26 00:00:00
+
+The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.
+His departure in 2019 was a consequence of the criticism he faced regarding his inappropriate behavior towards women, his hurtful comments about people with Down's Syndrome and his remarks relating to the Epstein scandal.
+His resignation came as a relief for many who have suffered his bad behavior.
```

Comment:
```suggestion
His resignation came as a relief for many who have suffered his actions.
```

Reading this again, we already used "behavior" in the sentence before and it reads a bit strange.

Reply:
I also was not really happy with this sentence but didn't have any alternative at hand. Jan's proposal is perfect and omits the term "bad".